src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1094,7 +1094,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
                 flags |= SF_TRUSTED;
             }
 
-            if (! app_.getOPs().getValidationPublicKey().size())
+            if (! app_.getValidationPublicKey().size())
```

Comment:
I'm wondering if having `bool Application::validating() const` makes sense.

Checking the size of the public key is a bit like checking whether your car radio is working by connecting a multimeter to the antenna input, instead of turning the radio on and seeing if sound comes out of the speakers.